pkg/parser/node/accessor.go

```diff
@@ -96,6 +97,30 @@ func (node *Node) GetStringMapping() (map[string]string, error) {
 	return result, nil
 }
 
+func (node *Node) GetFloat64Mapping() (map[string]float64, error) {
+	result := make(map[string]float64)
+
+	if _, ok := node.Value.(*MapValue); !ok {
+		return nil, node.ParserError("expected a map")
+	}
+
+	for _, child := range node.Children {
+		stringValue, err := child.GetStringValue()
```

Comment:
I wonder if this value should be expanded with `env`. What do you think?

Reply:
Enabled expansion https://github.com/cirruslabs/cirrus-cli/pull/522/commits/927638eda912b4609d590b2dce8602cc4e10edfe.